replica/rest/model.go

```diff
@@ -194,6 +202,7 @@ func NewReplica(context *api.ApiContext, state replica.State, info replica.Info,
 		actions["setrevisioncounter"] = true
 		actions["setreplicacounter"] = true
 		actions["updatecloneinfo"] = true
+		actions["setcheckpoint"] = true
```

Comment:
I don't think we need this in Rebuilding state.. we might also not needed in Dirty state..
Is Open the state of replica after it becomes RW? only that state need to be allowed

Reply:
As soon as an IO is written after opening the state becomes dirty.
It has been allowed in rebuilding state because this will be triggered as a part of verifyRebuildReplica for WO replica. While calling verifyRebuildReplica, replica is in Rebuilding state.